### DISCUSSION TITLE
Predictable layout for the game features

### DISCUSSION THREAD

**Discussion by denilsonsa** [IsThereAnyDeal/AugmentedSteam#1809] [Ideas]
We should have a "box" (a small grid) with all the important game features listed. In this grid, each "feature" will always be located at the same position, making it easy to know if the game has a feature or not.

## Motivation

This idea started up as a desire to know, at a glace, if a game has support for Steam Cloud saves or not. Mostly because I'm surprised and disappointed that [Cave Story](https://store.steampowered.com/app/200900/Cave_Story/) has no Steam Cloud save, which means I would need to start playing it from the start, instead of continuing from my older save from another computer.

But, then, I expanded the idea to the other game features.

## Problem

Every game has a list of "features" at the sidebar. However, the UI/UX for it is quite bad:

![image](https://github.com/IsThereAnyDeal/AugmentedSteam/assets/121676/df26953e-7443-4694-9571-f82f859d2e0c)

It's bad because you have to stop and read each item to figure out if it has or hasn't something. For instance, you might be tempted to believe that [this game](https://store.steampowered.com/app/440/Team_Fortress_2/), with such a long list, has everything:

![image](https://github.com/IsThereAnyDeal/AugmentedSteam/assets/121676/04763068-671f-4a5a-bac5-4617a9105188)

But if you pay attention, Steam Cloud isn't listed.

## Proposed solution

We could have something like this:

![image](https://github.com/IsThereAnyDeal/AugmentedSteam/assets/121676/fc22c9fd-d1fe-4e63-9891-3e24c1d3fedd)

At a glance, it's easy to see if a game has achievements, cards, or cloud save. It's also clear which games are only single-player, or only multiplayer, and also what kind of multiplayer (PvP or Co-op, Local or Online).

This is my first sketch, done quickly in Inkscape. We can certainly improve it…

* We could display **how many** achievements and how many trading cards in the label.
* We could add **more features**, such as Remote Play Together, Steam Leaderboards, In-App Purchases, Steam Workshop, VAC enabled, Level Editor, and anything else that might be relevant.
* We could try showing Keyboard/Mouse and Full/Partial Controller support, but this data may be difficult to obtain.
* We could try to display LAN multiplayer instead of Online multiplayer, for the few titles that still support LAN. This can either be a new item in the grid, or just a dynamic label explaining the kinds of multiplayer supported (e.g. LAN+Online).
* We could also extend this to **languages**. The user would select a few "favorite" languages in the extension options, and a few grid items would show up, clearly showing the each of the favorite languages is supported (in the interface, in the audio, in subtitles). This is useful for people who speak two or three languages. (e.g. Some countries have more than one official language or de-facto language; many people move and end up living in a foreign country; many people are just learning another language.)
* Similar to the languages, we could let the user pick certain **VR devices** and the grid items would show compatibility if said devices. (I don't own any VR gear, I don't know how it works.)

I think this neat, compact, and organized treasure of information should be displayed above the fold, or as the first thing at the top of the sidebar.

**Comment by candela97** [comment]
Appreciate the detailed proposal!

Enhancements to the "features" section looks doable with some CSS tweaks, though the info there is often unreliable, because Steam doesn't automatically add them and developers forget.

Other additional improvements would take quite a bit of work to get right, especially since they encompass several feature areas where Steam enforces different sets of rules.
